src/app/features/tech-record/components/tech-record-title/tech-record-title.component.html

```diff
@@ -1,4 +1,21 @@
 <dl *ngIf="currentTechRecord$ | async as currentTechRecord" class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Vehicle type</dt>
+    <dd id="vehicleType" class="govuk-summary-list__value">
+      {{ (currentTechRecord$ | async)?.vehicleType ?? vehicleType | defaultNullOrEmpty }}
```

Comment:
```suggestion
      {{ (currentTechRecord$ | async)?.vehicleType | defaultNullOrEmpty }}
```
See my comment about the getter in the component.

Reply:
removed..